Sources/BuildTool/BuildTool.swift

```diff
@@ -71,6 +72,31 @@ struct TestLibrary: AsyncParsableCommand {
     }
 }
 
+@available(macOS 14, *)
+struct GenerateCodeCoverage: AsyncParsableCommand {
```

Comment:
Isn't it the same as TestLibrary but with different params? Why dedicated structure then?